### ISSUE TITLE
Add 3d / depth effect on the second plan like with UIPresentationController

### ISSUE THREAD

**Issue by shouze** [stipsan/react-spring-bottom-sheet#145]
No idea of how to proceed at the moment, but would be nice to be able to reproduce iOS UIPresentationController effect on the background, like in iOS Mail composition or Apple Music ([see video here](https://www.reddit.com/r/reactnative/comments/aq7wz6/replicating_ios_compose_email_modal_in_reactnative/)).

**Comment by shouze** [comment]
Looks like this effect has been reproduced by https://github.com/Temzasse/react-modal-sheet

**Comment by stipsan** [comment]
Yeah, for this to be possible I need to expose the internal spring APIs. It’s a bit tricky but it’s on the roadmap :)

**Comment by shouze** [comment]
Tell me if and how we can help, impressed by the good job you've already done :muscle:.
We were prototyping our own drawer modal / bottom sheet when we found yours today and it's :heart: (we also chose to use `react-spring` and `react-use-gesture` for our prototype).

Regarding aria, I see you didn't choose to use `react-aria`, would you mind if we try to add it in a pull request?
